libclc/amdgcn-amdhsa/libspirv/math/log2.cl

```diff
@@ -0,0 +1,43 @@
+ #include <spirv/spirv.h>
+ #include <clcmacro.h>
+
+ #ifdef cl_khr_fp16
+ #pragma OPENCL EXTENSION cl_khr_fp16 : enable
+ #endif  
+
+ double __ocml_log2_f64(double);
+ float __ocml_log2_f32(float);
+ half __ocml_log2_f16(half);
```

Comment:
Should it be hidden behind an `ifdef`?

Also: below clang format and further down commented out code.